codec-http/src/main/java/io/netty/handler/codec/http/DefaultCookie.java

```diff
@@ -19,141 +19,117 @@
 import java.util.Set;
 import java.util.TreeSet;
 
-
-
 /**
  * The default {@link Cookie} implementation.
+ *
+ * @deprecated Use {@link io.netty.handler.codec.http.cookie.DefaultCookie} instead.
  */
-public class DefaultCookie implements Cookie {
+@Deprecated
+public class DefaultCookie extends io.netty.handler.codec.http.cookie.DefaultCookie implements Cookie {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2176)